### PR TITLE
Add a FAQ item on creating milestones

### PR DESCRIPTION
**Pull request by PeterWhittaker**
It took me a while of reading through the source to puzzle out that milestones can be created with the "-s open" option. This is not exactly clear, so having a FAQ entry seemed reasonable.

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -136,6 +136,17 @@ config (or export `$GHI_EDITOR` in your environment).
 $ git config --global ghi.editor vim
 ```
 
+- __How do I create a milestone?__
+
+Invoke `ghi milestone` with the `-s` option:
+
+``` sh
+$ git milestone -s open
```
Comment (stephencelis) [inline README.markdown]: I think you meant `ghi milestone` here?

**Comment by stephencelis** [comment]
Hm, do you think the description text in `ghi milestone help` is not sufficient?

**Comment by PeterWhittaker** [comment]
Stephen, thanks for the quick reply. No, the help doesn't make it clear and
nor is it obvious that "-s open/close" actually changes status (but I'm
glad I went through this because then it "clicked" when I went to close an
issue, which uses the same toggle).

Likewise, it wasn't obvious that "edit iss# -M" removes the milestone added
by "edit iss# -M ms#".

I think part of the problem is that there are toggles (-s), sets, and gets,
and it ain't always clear which is which.

Thanks again,


Peter Whittaker
EdgeKeep Inc.
www.edgekeep.com
+1 613 864 5337
+1 613 864 KEEP

**Comment by stephencelis** [comment]
Going to close this one for now. I'm open to documenting it but my comment about the `git` typo wasn't addressed. Feel free to chime in here again if you force-push an update, thanks!

**Pull request closed without merging**